tests/integration/targets/nxos_acl_interfaces/tests/cli/populate_config.yaml

```diff
@@ -1,7 +1,16 @@
 ---
 - name: Adding base configuration
-  ansible.netcommon.cli_config:
-    config: "ip access-list ACL1v4\nip access-list NewACLv4\nipv6 access-list ACL1v6\n\
-      ipv6 access-list NewACLv6\nip access-list PortACL\ninterface Ethernet1/2\n\
-      \  ipv6 traffic-filter ACL1v6 in\ninterface Ethernet1/5\n  ip port access-group\
-      \ PortACL in\n  ip access-group ACL1v4 out\n  ipv6 traffic-filter ACL1v6 in\n"
+  cisco.nxos.nxos_command:
+    commands:
+      - "configure"
```

Comment:
I don't like this vs having two nxos_config plays, but if this is appreciably faster than that (especially over the number of times we run it) then I guess it's fine.

I would however add a comment describing the problem (and link to an issue tracking it?)

Reply:
Thinking out loud, the time taken by the extra tasks is probably very small in the grand scheme of things and using `src` for remove_config turned out to be risky in some cases leading to leftovers in config that cause assertions to fail. This is mostly because NX-OS throws an error if you try to remove an ACL that is not already configured on an interface and as a result, the subsequent negate lines are not sent to the appliance.